phpseclib/File/ANSI.php

```diff
@@ -416,28 +372,22 @@ function appendString($source)
                                                 case 47: $back = 'white'; break;
 
                                                 default:
-                                                    user_error('Unsupported attribute: ' . $mod);
+                                                    //user_error('Unsupported attribute: ' . $mod);
```

Comment:
I commented it out because, honestly, trying to figure out what all the ANSI escape codes do is difficult. Some of them I just do `break` and that's it because they don't apply (eg. show the cursor position, etc) but others...  I'm not sure what they do. I don't want to add a `case` statement for them that I `break` since I don't know what they do but neither do they seem necessary to create the desired visual effects.

I didn't delete the line because it could still be useful for diagnostic purposes.